index.js

```diff
@@ -0,0 +1,115 @@
+const ethers = require('ethers')
+// Is the voteParams same for all/some schemes of a common?
+
+// TODO: Edit constants/ Make them function params
+const DAOFactoryInstance = "0x332280007303d5942dd60A945169b2b303320686"
```

Comment:
maybe add a call to get DAOFactory address/instance

Reply:
What do you mean by add API for that?
I can export it if that's what you mean...

Reply:
yes, export it as it can be read from outside if needed.
in general the address of it should be  called from subgraph .
can you add this option maybe ?

Reply:
So it's better to have it as a parameter from Common as it is already querying the subgraph.